qe/tests/org.jboss.tools.fuse.qe.reddeer.tests/launcher/RedDeer Test Linux x86_64.launch

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
shouldn't we named it RedDeer Test? it is taking the whole set of available plugins so no difference on the platform unless I missed a configuration?

Reply:
Good point! This was changed in recent commit and I didn't notice that :( I will prepare another launch config.